src/components/ChainInformation.tsx

```diff
@@ -54,7 +54,7 @@ export const ChainInformation = () => {
 
         <ChainInfoCard
           title={t('CHAIN.CHAININFORMATION.totalBatchesCommitted')}
-          description={getDescription(chainData?.batchesInfo?.commited)}
+          description={getDescription(Number(chainData?.batchesInfo?.commited), notAvailable)}
```

Comment:
These `Number()` are necessary?

Reply:
The backend is providing a string type for batchesInfo